### PR TITLE
Set image id as unique one

### PR DESCRIPTION
**Pull request by kakaZhou719**
### Describe how you did it
link to #24
set image ID as unique one

### CODE REVIEW

File: parser/parser.go [hunk 1]
```diff
@@ -38,14 +39,14 @@ func (p *Parser) Parse(kubeFile []byte, name string) *v1.Image {
...
-
-		//TODO count layer hash
 		image.Spec.Layers = append(image.Spec.Layers, v1.Layer{
 			Hash:  "",
 			Type:  layerType,
 			Value: layerValue,
 		})
 	}
+	// gen consistent image ID
+	image.Spec.ID = utils.MD5(kubeFile, []byte(name))
```
Comment (justadogistaken) [inline parser/parser.go:49]: curious why generate id base on kubefile and image name instead of utils.GenUniqueID(32)

Comment (fanux) [reply parser/parser.go:50]: This is not appropriate and may even lead to bugs if two different image using same Clusterfile. Image ID should not using Clusterfile MD5

Comment (oldthreefeng) [reply parser/parser.go:50]: like rootfs image. it can be affect only the kubernetes version. not the others things.

**Review by oldthreefeng — COMMENTED** [review]
so we must find the affect things of each image layer.

File: parser/parser.go (hunk 1 above)
Comment (kakaZhou719) [reply parser/parser.go:50]: image id = md5sum(clusterfile+image name), same clusterfile with same image name guess that is a same image . what do you think ?@all

Comment (fanux) [reply parser/parser.go:50]: Not good..

Comment (justadogistaken) [reply parser/parser.go:50]: docker generate image id refer to content of image config. Is it fine to calculate image id depends on image metadata?

**Pull request closed without merging**